test/fit_tests/deploy/rackhd_source_install.py

```diff
@@ -41,19 +46,15 @@ def test01_install_rackhd_dependencies(self):
         self.assertEqual(fit_common.remote_shell("git config --global http.sslverify false")['exitcode'], 0, "Git config failure.")
         self.assertEqual(fit_common.remote_shell("git config --global http.proxy " + fit_common.GLOBAL_CONFIG['repos']['proxy']
                                                   )['exitcode'], 0, "Git proxy config failure.")
```

Comment:
Might want the "apt-get -y update" before "apt-get -y install git".  Generally, should update the package metadata before installing any of the packages.

Is the dist-upgrade necessary?  If not, it may save quite a bit of time if we don't need to do this.